params/extras/precompile_upgrade_test.go

```diff
@@ -15,8 +16,7 @@ import (
 
 func TestVerifyUpgradeConfig(t *testing.T) {
 	admins := []common.Address{{1}}
-	chainConfigCpy := Copy(TestChainConfig)
-	chainConfig := GetExtra(&chainConfigCpy)
+	chainConfig := &ChainConfig{}
```

Comment:
note I tried to use the empty ChainConfig (former ChainConfigExtra) where possible
We can switch back to TestChainConfig if others prefer.
Less test setup seems preferred to me, but we could keep it to avoid any risk here.